src/main/java/use_cases/PostManager.java

```diff
@@ -0,0 +1,73 @@
+package use_cases;
+
+import entities.*;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+/*
+store all the posts existing in the app,
+add a new post, comment/like a Post
+ */
+public class PostManager {
+    private ArrayList<Post> posts;
+
+    public PostManager(){
+        this.posts = new ArrayList<Post>();
+    }
+    //in case
+    /*
+        Returns true if the interaction could be done, false otherwise
+        if the type is true, then the interaction is a comment
+     */
+    private boolean interactPost(Post post, Object obj, boolean type){
+        // it looks into all the post in the stored structure
+        // in order to work, the comparison of post should be
+        // implemented
+        for(int i = 0; i < this.posts.size(); i++)
+        {
+            if(this.posts.get(i).equals(post))
+            {
+                if(type)
+                {
+                    this.posts.get(i).addComment((Comment) obj);
+                }
+                else
+                {
+                    this.posts.get(i).addLike((User) obj);
+                }
+                return true;
+
+            }
+        }
+        return false;
+    }
+
+    public Ingredient createIngridient(String pName){
```

Comment:
What does pName stand for? Could we maybe replace this with "name" or "ingredientName" for clarity?

Reply:
Hold on, I think we need to differentiate between measurable ingredients and countable. For measurable ingredient, I'd like to pass in Strings that look like this: "20 grams sugar". Could you please do createMeasurableIngredient(String ingredient) that manages that input? For countable ingredients, I'd like to pass in input that look like: "1 apple." Could you please make another method that is createCountableIngredient(String ingredient)? Let me know if that makes sense.

Reply:
@sebastianibam002 If you think this is too much work, I think you would be able to implement this later past the Phase 0 deadline and just focus on getting Post creation and commenting/liking working for now. At this point our number one priority is getting code that works with our walkthrough